docs/gen-ai/non-normative/models.ipynb

```diff
@@ -222,6 +239,34 @@
     "# Print the JSON schema for the SystemInstructions model\n",
     "print(json.dumps(SystemInstructions.model_json_schema(), indent=4))"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f019c33a",
+   "metadata": {},
+   "source": [
+    "## `gen_ai.tool.definitions` model\n",
+    "\n",
+    "Corresponding attribute: [`gen_ai.tool.definitions`](/docs/registry/attributes/gen-ai.md#gen-ai-tool-definitions).\n",
+    "JSON schema: [`gen_ai-tool-definitions.json`](../gen-ai-tool-definitions.json)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a9e84726",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class ToolDefinitions(RootModel[List[ToolDefinition]]):\n",
```

Comment:
I think ToolDefinition should be defined in this cell since it's not used elsewhere